packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -791,6 +827,10 @@ class GenerationInfoFrame {
     getUniqueLambdaBlockName(lambda: SdsBlockLambda): string {
         return `${BLOCK_LAMBDA_PREFIX}${this.blockLambdaManager.assignId(lambda)}`;
     }
+
+    isInsidePipeline(): boolean {
+        return this.insidePipeline;
+    }
```

Comment:
You can also rename the property `insidePipeline` to `isInsidePipeline` and make that public. The getter does not add much here. Up to you, though.